src/gui.js

```diff
@@ -8502,6 +8502,35 @@ SpriteIconMorph.prototype.copySound = function (sound) {
     this.object.addSound(dup.audio, dup.name);
 };
 
+// SpriteIconMorph flashing
+
+SpriteIconMorph.prototype.flash = function () {
+    var previousOutlineColor = this.outlineColor.copy(),
```

Comment:
hmm... do we really need to copy the color? Can't we just remember it, or is it mutated anywhere in between here and switching back?

Reply:
We don't need to copy it indeed. I don't know why I thought we di.